flexeval/core/language_model/openai_api.py

```diff
@@ -108,7 +136,10 @@ def batch_complete_text(
                 **kwargs,
             ),
         )
-        return [res.choices[0].message.content for res in api_responses]
+        completions = [res.choices[0].message.content for res in api_responses]
+        if all(completion == "" for completion in completions):
+            logger.warning("All generated texts are empty string. Something may go wrong.")
```

Comment:
Just to fix a small grammar point and simplify the text.

```suggestion
            logger.warning("All generated texts are empty strings. Something may be wrong.")
```